subprojects/packagefiles/jbigkit/libjbig/meson.build

```diff
@@ -0,0 +1,14 @@
+sources = files('jbig.c', 'jbig_ar.c')
+sources85 = files('jbig85.c', 'jbig_ar.c')
+
+libjbig = library('jbig', sources)
+libjbig_dep = declare_dependency(
+    include_directories: include_directories('.'),
+    link_with: libjbig)
+meson.override_dependency('libjbig', libjbig_dep)
```

Comment:
... yet override_dependency is used, which is a relatively new API.

If, instead of `dependency_names = libjbig`, you used `libjbig = libjbig_dep`, then the meson.build file becomes backwards compatible with lots of older versions of meson, including one or two versions that support this alternative wrap-file syntax but not the dependency_names format.